blog/_posts/2016-08-20-hello-from-team-Crackers.md

```diff
@@ -0,0 +1,53 @@
+
+
+#### **NIKITA**
+I am currenty studying in 3rd year(undergrad) for the course Information and Communication Technology. I started coding with basic C and HTML since my high school. Later I got to know about web development, attended various workshops and discussions which acted as an ignition to working with web applications. Then I started studying various online courses and making informal projects. Later last summer I had my internship on educational web applications where I met very great people who taught me advance techniques in this field which is quite helpful for the project. In my free time I go out and play football with my team or fiddle around arts and crafts. Also I like travelling and mountain hiking.
+
+![Team Crakers](/img/blog/2016/teamCrackers2.jpg)
+
+### **ABOUT OUR PROJECT**
+
+WeCare '[WeCare](https://github.com/hkasera/WeCare "WeCare")'is a digital platform that will help NGOs to get onboard with technology to promote their campaigns. It will also help people in discoverability of various campaigns as well as provide detailed information about them. Volunteers can also contribute towards ongoing campaigns, run by different NGOs, via this platform. The best part about the platform is the ease of use and its effectiveness so that anyone can easily promote and market their campaigns to increase their visibility. It is a free service. We plan to create a console for NGO's once they register on the website to manage their campaigns.
+
+### **WHAT WE HAVE ACHIEVED SO FAR?**
+
+During initial days there was a bit miscommunication between the team and the mentor, as she was a bit intotravelling and we were taking some time in understanding the project.But once she was back, we arranged calls every week discussed on 2-3 issues and how should we approach them. Our first issue was which we decided to take a basic one is login/signup page. First we made it having two different pages but then we planned the structure and layout of the whole website nd figured out that merged tabular look would be much better with passport in its back end part. Second task which we are working on is a console or portal type where the NGO's can register their website or add upcoming campaigns for other people in world to view them or publish new information they get. For addressing this issue we went through some advance topics in node and also we planned out a layout for console. We are done with a parrallax style home page for WeCare, added an "About Us" page to it with its content and partially done with console task.
+
+
+### **What are our future goals?**
+
+Our present focus is on console task as it cannot afford to bring or have any sort of error because that would result in problems in publishing any information live. Later we will focus on following tasks provided by mentor:
+	- Update API Documentation
```

Comment:
if you want to show a list (with bullet points) here, it needs to be on a new paragraph (no need for a `<p>` tag, a simple newline break in your editor is enough). If you're okay with just showing the `-` as a "bullet point", that's okay too, but keep in mind that each list item needs two spaces (or more) at the end of it to create a line break :)